### PR TITLE
added docs for elasticsearch and opensearch for development Tab.

### PR DESCRIPTION
**Pull request by 07Himank**
I worked on ... because ... added docs for elasticsearch and opensearch for development Tab.

#
### Type of change:

- [ x] Improvement

### CODE REVIEW

File: openmetadata-docs/content/v1.1.0/deployment/bare-metal/index.md [hunk 1]
```diff
@@ -174,6 +174,10 @@ We support
 - Amazon OpenSearch (ElasticSearch) engine version upto 7.1 or Amazon OpenSearch engine version upto 1.3
 - Amazon RDS (PostgreSQL) engine version between 12 and 14.6
 
+Note:-
+    When using AWS Services the SearchType Configuration for elastic search should be opensearch, for both cases ElasticSearch and OpenSearch,
+as you can see in the ElasticSearch configuration example. 
+
```
Comment (akash-jain-10) [inline openmetadata-docs/content/v1.1.0/deployment/bare-metal/index.md:180]:
```suggestion
Note:-
    When using AWS Services the SearchType Configuration for elastic search should be `opensearch`, for both cases ElasticSearch and OpenSearch,
as you can see in the ElasticSearch configuration example below. 

```

File: openmetadata-docs/content/v1.1.0/deployment/docker/index.md [hunk 2]
```diff
@@ -229,6 +230,10 @@ We support
 - Amazon OpenSearch (ElasticSearch) engine version upto 7.10 or Amazon OpenSearch engine version upto 1.3
 - Amazon RDS (PostgreSQL) engine version between 12 and 14.6
 
+Note:-
+    When using AWS Services the SearchType Configuration for elastic search should be opensearch, for both cases ElasticSearch and OpenSearch,
+as you can see in the ElasticSearch configuration example.
+
```
Comment (akash-jain-10) [inline openmetadata-docs/content/v1.1.0/deployment/docker/index.md:236]:
```suggestion
Note:-
    When using AWS Services the SearchType Configuration for elastic search should be `opensearch`, for both cases ElasticSearch and OpenSearch,
as you can see in the ElasticSearch configuration example below. 

```

File: openmetadata-docs/content/v1.1.0/deployment/kubernetes/index.md [hunk 3]
```diff
@@ -235,6 +235,10 @@ We support
 - Amazon OpenSearch (ElasticSearch) engine version upto 7.10 or Amazon OpenSearch engine version upto 1.3
 - Amazon RDS (PostgreSQL) engine version between 12 and 14.6
 
+Note:-
+    When using AWS Services the SearchType Configuration for elastic search should be opensearch, for both cases ElasticSearch and OpenSearch,
+as you can see in the ElasticSearch configuration example.
+
```
Comment (akash-jain-10) [inline openmetadata-docs/content/v1.1.0/deployment/kubernetes/index.md:241]:
```suggestion
Note:-
    When using AWS Services the SearchType Configuration for elastic search should be `opensearch`, for both cases ElasticSearch and OpenSearch,
as you can see in the ElasticSearch configuration example below. 

```

**Review by akash-jain-10 — CHANGES_REQUESTED** [review]

File: openmetadata-docs/content/v1.1.0/deployment/bare-metal/index.md (hunk 1 above)
Comment (07Himank) [reply openmetadata-docs/content/v1.1.0/deployment/bare-metal/index.md]: Handled in latest commit

File: openmetadata-docs/content/v1.1.0/deployment/docker/index.md (hunk 2 above)
Comment (07Himank) [reply openmetadata-docs/content/v1.1.0/deployment/docker/index.md]: Handled in latest commit

File: openmetadata-docs/content/v1.1.0/deployment/kubernetes/index.md (hunk 3 above)
Comment (07Himank) [reply openmetadata-docs/content/v1.1.0/deployment/kubernetes/index.md]: Handled in latest commit

**Review by akash-jain-10 — APPROVED** [review]

**Pull request merged**